src/NXP/Classes/Token/TokenDivision.php

```diff
@@ -47,7 +47,7 @@ public function execute(&$stack)
     {
         $op2 = array_pop($stack);
         $op1 = array_pop($stack);
-        $result = $op1->getValue() / $op2->getValue();
+        $result = $op2->getValue() > 0 ? $op1->getValue() / $op2->getValue() : 0;
```

Comment:
Hello! Thank for your work! What about negative second operator? And why zero result, mb infinity?